lib/vmpooler/pool_manager.rb

```diff
@@ -258,7 +261,8 @@ def clone_vm(template, folder, datastore, target)
         $redis.decr('vmpooler__tasks__clone')
 
         begin
-          $graphite.log($config[:graphite]['prefix'] + ".clone.#{vm['template']}", finish) if defined? $graphite
+          $statsd.timing($config[:statsd]['prefix'] + ".clone.#{vm['template']}", finish) if $statsd
```

Comment:
And I think we want to always presume statsd is available, and unify statsd/graphite into one thing if possible. Fire and forget. If it's botched then log/warn (inside our wrapper class), but carry on and not interrupt the caller.

In contexts where it made sense we often just published some data via activesupport notifications or whatever and stats became a listener; not recommended here.